### ISSUE TITLE
Rotation problem...

### ISSUE THREAD

**Issue by Tristan79** [billw2/pikrellcam#20]
I have a problem with rotation. I wanted to use this to monitor a door...  so I placed the camera like I though was the best view..., thinking I could rotate the image:

![screen shot 2016-12-29 at 23 43 05](https://cloud.githubusercontent.com/assets/4981984/21556493/26b70694-ce21-11e6-98a6-8d0c170e71b0.png)

so i edited /home/pi/.pikrellcam/pikrellcam.conf and at the end I placed rotation 90 still thinking it would come out fine. The camera has not moved inbetween...

![screen shot 2016-12-29 at 23 46 10](https://cloud.githubusercontent.com/assets/4981984/21556526/8a5d831c-ce21-11e6-8439-5310edc8c794.png)

but... when it rotates, it cuts off the parts I want!? And adds new parts...  it keeps the same tv aspect ratio :-/

is it possible in any way such that the view in the first image (although good for now, but with a  90 degree crooked neck when viewing) is somehow rotated on the webpage (and recording) to 1080x1920?

**Comment by c765f614** [comment]
Coming back to the above situation, I got almost the same problem. After a few days trying to solve it, no solutions found yet.

Problem is that the visible section is 1920x1080 (1080p) but I want a visibility of 1080x1920 (so the visibility 90deg rotated, so I can see what I need.
Is there or will there be a solution for this?

Many thanks..

**Comment by creeperbait** [comment]
Yeah, I have EXACTLY the same situation; trying to capture the whole of a door, which tend to be, you know, verticalish.

**Comment by cwdosman** [comment]
I just tried it on the fly, but when you change the resolution in /home/pi/.pikrellcam/pikrelcam.conf the video width to 1080 and the height to 1920, the preview was "verticalish". I didnt recorded a video but also there it should work.

**Comment by CycleMark** [comment]
The settings you want are at the end of the file. Look for "rotation" 

![PikrelCamImageRotationSettings](https://user-images.githubusercontent.com/27418725/56088927-1c981880-5e82-11e9-9e8f-59b13e35d692.PNG)


Mark

**Comment by creeperbait** [comment]
cwdosman, thanks! It's taken me a while to get to it, but that's exactly it, along with the corresponding still_width and still_height.
